Dockerfile

```diff
@@ -12,13 +12,15 @@ RUN add-apt-repository -y ppa:snappy-dev/tools-proposed && \
 # install dependencies
 RUN apt-get update && apt-get install -qy \
   jenkins-launchpad-plugin \
-  snappy-tests-job && \
+  snappy-tests-job \
+  jq && \
   rm -rf /var/lib/apt/lists/*
 
 # copy scripts
 COPY scripts/authentication.groovy \
   scripts/jobs.groovy \
-  /usr/share/jenkins/ref/init.groovy.d/
+  /usr/share/jenkins/ref/init.groovy.d/ \
```

Comment:
It seems that the order should be reverted:

scripts/poll_spi_result.groovy \
/usr/share/jenkins/ref/init.groovy.d/

being the latter the destination folder. More about the .groovy extension below :)